### DISCUSSION TITLE
Erratum correcting treatment info but without a nomenclature heading

### DISCUSSION THREAD

**Discussion by Carol-Sokolowicz** [plazi/community#75] [Q&A]
Hi Donat,
This Erratum from Zootaxa is correnting an info of the treatment, however, the treatment heading was not provided on erratum. How should I proceed since I cannot mark a treatment without a nomenclature heading.

![image](https://user-images.githubusercontent.com/57491712/78148486-49cf3500-740b-11ea-87e7-ab65b8888b84.png)

**It also brings corrections of the capition, but does not provide the figures**

![image](https://user-images.githubusercontent.com/57491712/78148596-67040380-740b-11ea-83c6-409fe1064b89.png)

Here is the PDF of the erratum: https://drive.google.com/open?id=1gupyZUr9UYc37gBos4u41MCASE9u1Esf

Here is the UUID of the original article: 7F76C14CFFC69477FFC32E1DFFD0C82F

and the UUID of the treatment the erratum corrects: 834FB934FFC49470FF542F6CFA4ACADC

**Comment by mguidoti** [comment]
@myrmoteras ?

**Comment by mguidoti** [comment]
@myrmoteras we also had a fairly recent discussion [about how EJT should produce their errata layouts here](https://github.com/plazi/plazi-at-poa/issues/480). 

I think this is important as we are suggesting guidelines to enable automate text-mining and data linking on errata. We could even make some noise about it.

Have you had the time to check this up?

Best,

**Comment by flsimoes** [comment]
@myrmoteras @mguidoti checking up on this. Should I move it to discussions?

**Comment by mguidoti** [comment]
Yes, I would say so. Also, let's wait @myrmoteras input.

Thanks
